apps/dashboard/src/composables/useUserType.js

```diff
@@ -25,16 +38,22 @@ export default function useUserType(userClaims) {
       return AUTH_USER_TYPE.SUPER_ADMIN;
     }
 
-    if (claims?.role === UserRoles.LAUNCH_ADMIN) {
-      return AUTH_USER_TYPE.LAUNCH_ADMIN;
-    }
+    if (claims?.role) {
+      if (claims?.role === UserRoles.LAUNCH_ADMIN) {
+        return AUTH_USER_TYPE.LAUNCH_ADMIN;
+      }
```

Comment:
```suggestion
const roleTypeMap = {
  [UserRoles.LAUNCH_ADMIN]: AUTH_USER_TYPE.LAUNCH_ADMIN,
  [UserRoles.ADMIN]: AUTH_USER_TYPE.ADMIN,
}
if (claims?.role && roleTypeMap[claims.role]) {
  return roleTypeMap[claims.role];
```
We can remove the nested if-statement, and allow for a constant which can be expanded upon later.